src/main/scala/org/zalando/spearheads/innkeeper/routes/GetRoutes.scala

```diff
@@ -60,10 +60,12 @@ class GetRoutes @Inject() (
                 case _ => None
               }.toList
 
-              logger.debug(s"Filters $filters. Embed: $embed")
+              val pagination = None
```

Comment:
Get `page` and `per_page` and create a `Pagination`. Maybe we can use http://doc.akka.io/docs/akka-http/current/scala/http/routing-dsl/directives/parameter-directives/parameters.html